mds/migrations/0044_compliance.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.1.11 on 2019-09-09 14:34
+
+import django.contrib.postgres.fields.jsonb
+import django.core.serializers.json
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mds', '0043_policy'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Compliance',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, primary_key=True, serialize=False)),
+                ('rule', models.UUIDField(default=uuid.uuid4)),
+                ('geography', models.UUIDField(default=uuid.uuid4)),
+                ('start_date', models.DateTimeField()),
+                ('end_date', models.DateTimeField(blank=True, null=True)),
+            ],
+        ),
+        migrations.AlterField(
```

Comment:
You may clean the migration if you want : the following AlterField are not mandatory (since not added in models.py) 
It is no mandatory though, I tend to do it because my commit will be cleaner :)